config/development.yaml

```diff
@@ -55,6 +55,7 @@ featureToggles:
   enableCaseTrainTrack: true  
   enableRAComponent: true
   enablePcqComponent: true
+  enableC100CaseProgressionTrainTrack: true
```

Comment:
why is this flag present for other people confidentiality?

Reply:
Removed